e2e/setup/serializers/image/image-processor.ts

```diff
@@ -30,7 +30,7 @@ export class SnapshotDataProcessor {
   public static async process(context: jest.MatcherContext, received: Buffer): Promise<SnapshotData> {
     const {testPath, currentTestName} = context;
     mkDir(SnapshotDataProcessor.SNAPSHOT_DIR);
-    const testName = toKebabCase(`${testPath!.split('/').pop()} ${currentTestName}`);
+    const testName = buildSnapshotName(testPath!.split('/').pop()!, currentTestName!);
```

Comment:
We talk about usage subdirecories including file name. 
So for
```
 tests
   subdir
     test.file.ts/girkin:
       - Test 1
       - Test 2
```
The output should be
```
 tests
   __snaoshots__
      subdir
        test.file
          test-1-snap.jpg
          test-2-snap.jpg
```
```
 .diff
    subdir
      test.file
        test-1-snap.jpg
        test-2-snap.jpg
```